src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -1023,31 +1022,12 @@ def test_loop_independent_iterations(monkeypatch, dist_mem):
     loops = schedule.walk(DynLoop)
     assert not loops[0].independent_iterations()
     assert loops[1].independent_iterations()
-    # Loop over dofs.
-    _, invoke_info = parse(os.path.join(BASE_PATH,
-                                        "15.1.6_aX_plus_bY_builtin.f90"),
-                           api=TEST_API)
-    psy = PSyFactory(TEST_API, distributed_memory=dist_mem).create(invoke_info)
-    schedule = psy.invokes.invoke_list[0].schedule
-    loop = schedule.walk(DynLoop)[0]
-    assert loop.independent_iterations()
-    # Test that we can get hold of any DA messages if we supply our own
-    # instance of DependencyTools.
-    dtools = DependencyTools()
-    loop.independent_iterations(dep_tools=dtools)
-    assert len(dtools.get_all_messages()) == 1
     # Unsupported/unknown loop type
-    monkeypatch.setattr(loop, "_loop_type", "broken")
+    monkeypatch.setattr(loops[1], "_loop_type", "broken")
     with pytest.raises(InternalError) as err:
-        loop.independent_iterations()
+        loops[1].independent_iterations()
     assert "loop of type 'broken' is not supported" in str(err.value)
     monkeypatch.undo()
-    # Test that DA warnings about a scalar variable are ignored if the loop is
-    # over DoFs.
-    monkeypatch.setattr(DependencyTools, "get_all_messages",
-                        lambda _1: [Message("just a test",
-                                            DTCode.WARN_SCALAR_REDUCTION)])
-    assert loop.independent_iterations()
     # Test when the DA returns True. Since this currently never happens for an
```

Comment:
I tried and we miss a line if we don't do it. I think the key thing here is that the LFRic kernel is over cell columns rather than dofs. I've extended the comment.